Adds in an example to show how you can use the values inside a manifest (#6957)
umbraco/UmbracoDocs: 15/umbraco-cms/customizing/foundation/localization.md

<umb-localize key="section_numberOfItems" args="[5]"></umb-localize>
```

### Using with manifests

> You can localize values in a manifest, for example the name of the dashboard tab that is visible in the UI, by prefxing the value with a #.

Reviewer: ```suggestion
You can localize values in a manifest. For example, prefix the name of the dashboard tab visible in the UI with a `#`.
```

> For example,a manifest registering a dashboard with `umbraco-package.json` or JavaScript can localize the `label` property in the `meta` object like this.

Reviewer: ```suggestion

#### Example
A manifest registering a dashboard with `umbraco-package.json` or JavaScript can localize the `label` property in the `meta` object like this.
```

> For example a manifest for registering a dashboard with umbraco-package.json or via JavaScript can localize the label property in the meta object like so.

Reviewer: ```suggestion
For example,a manifest registering a dashboard with `umbraco-package.json` or JavaScript can localize the `label` property in the `meta` object like this.
```
